mindee/cli.py

```diff
@@ -70,47 +72,65 @@ class CommandConfig(Generic[TypeDoc]):
         help="FR Bank Account Details",
         doc_class=documents.fr.TypeBankAccountDetailsV1,
     ),
+    "invoice-splitter": CommandConfig(
+        help="Invoice Splitter",
+        doc_class=documents.TypeInvoiceSplitterV1,
+        is_sync=False,
+        is_async=True,
+    ),
 }
 
 
-def _get_input_doc(client, args) -> DocumentClient:
-    if args.input_type == "file":
-        with open(args.path, "rb", buffering=30) as file_handle:
-            return client.doc_from_file(input_file=file_handle)
-    elif args.input_type == "base64":
-        with open(args.path, "rt", encoding="ascii") as base64_handle:
-            return client.doc_from_b64string(
-                input_string=base64_handle.read(), filename="test.jpg"
-            )
-    elif args.input_type == "bytes":
-        with open(args.path, "rb") as bytes_handle:
-            return client.doc_from_bytes(
-                input_bytes=bytes_handle.read(), filename=bytes_handle.name
-            )
-    elif args.input_type == "url":
-        return client.doc_from_url(url=args.path)
-    return client.doc_from_path(args.path)
+def _get_input_doc(
+    client: Client, args: Namespace, parsed_path: Optional[str] = None
+) -> DocumentClient:
+    if not parsed_path:
+        if args.input_type == "file":
+            with open(args.path, "rb", buffering=30) as file_handle:
+                return client.doc_from_file(input_file=file_handle)
+        elif args.input_type == "base64":
+            with open(args.path, "rt", encoding="ascii") as base64_handle:
+                return client.doc_from_b64string(
+                    input_string=base64_handle.read(), filename="test.jpg"
+                )
+        elif args.input_type == "bytes":
+            with open(args.path, "rb") as bytes_handle:
+                return client.doc_from_bytes(
+                    input_bytes=bytes_handle.read(), filename=bytes_handle.name
+                )
+        elif args.input_type == "url":
+            return client.doc_from_url(url=args.path)
+        return client.doc_from_path(args.path)
+    return client.doc_from_url(parsed_path)
```

Comment:
There should be only one condition to return `doc_from_url`, or am I missing something?

In fact it does not seem like you are calling this function with the argument `parsed_path` set.